internal/provider/airflow_cluster_resource.go

```diff
@@ -0,0 +1,870 @@
+				PlanModifiers:       []planmodifier.Object{objectplanmodifier.UseStateForUnknown()},
+				MarkdownDescription: "Public connection info",
+			},
+			//"custom_remote_connection_info": schema.SingleNestedAttribute{
+			//	Computed:            true,
+			//	Attributes:          airflowCustomRemoteConnectionInfoResSchema(),
+			//	PlanModifiers:       []planmodifier.Object{objectplanmodifier.UseStateForUnknown()},
+			//	MarkdownDescription: "Remote container registry connection information.",
+			//},
+		},
+		Blocks: map[string]schema.Block{
+			"resources": schema.SingleNestedBlock{
+				Description: "Cluster resources",
+				Blocks: map[string]schema.Block{
+					"airflow": schema.SingleNestedBlock{
+						Attributes: map[string]schema.Attribute{
+							"max_worker_count": schema.Int64Attribute{
+								Required:            true,
+								MarkdownDescription: "Maximum number of Airflow workers.",
+								Validators: []validator.Int64{
+									int64validator.AtLeast(1),
+									int64validator.AtMost(10),
+								},
+							},
+							"min_worker_count": schema.Int64Attribute{
+								Required:            true,
+								MarkdownDescription: "Minimum number of Airflow workers.",
+								Validators: []validator.Int64{
+									int64validator.AtLeast(1),
+									int64validator.AtMost(10),
+								},
+							},
+							"environment_flavor": schema.StringAttribute{
+								Required:            true,
+								MarkdownDescription: "Environment flavor for Airflow.",
+								Validators: []validator.String{
+									stringvalidator.OneOf("small", "medium", "large"),
+								},
+							},
+							"worker_concurrency": schema.Int64Attribute{
+								Required:            true,
+								MarkdownDescription: "Concurrency level for Airflow workers.",
+								Validators: []validator.Int64{
+									int64validator.AtLeast(1),
+									int64validator.AtMost(30),
+								},
+							},
+							"worker_disk_size": schema.Int64Attribute{
+								Required:            true,
+								MarkdownDescription: "Disk size for Airflow workers.",
+								Validators: []validator.Int64{
+									int64validator.AtLeast(1),
+									int64validator.AtMost(10),
+								},
+							},
+							"worker_preset": schema.StringAttribute{
+								Required:            true,
+								MarkdownDescription: "Worker preset configuration.",
+								Validators: []validator.String{
+									stringvalidator.OneOf("small", "medium", "large"),
+								},
+							},
+						},
+					},
+				},
+			},
+			"access": AccessSchemaBlock(),
+			"config": schema.SingleNestedBlock{
+				Description: "Cluster configuration",
+				Attributes: map[string]schema.Attribute{
+					"version_id": schema.StringAttribute{
+						Required:            true,
+						MarkdownDescription: "Version ID of the Airflow cluster.",
+					},
+					"custom_image_digest": schema.StringAttribute{
+						Optional:            true,
+						MarkdownDescription: "Custom image digest for the Airflow cluster.",
```

Comment:
this 2 fields acts as one-of, it's either custom image or requirement txt, let's add this do comment.